R/epidist.R

```diff
@@ -333,11 +333,19 @@ validate_epidist <- function(epidist) {
       c("disease", "epi_dist", "prob_dist", "uncertainty", "summary_stats",
         "citation", "metadata", "method_assess", "notes") %in%
       attributes(epidist)$names,
-    "Epidist must contains a disease (single character string)" =
+    "epidist must contains a disease (single character string)" =
```

Comment:
```suggestion
    "epidist must contain a disease (single character string)" =
```

Reply:
Resolved in 0ad700bc6a040c92f3489e40b493cc540780c0a4.